test/forms/RegisteredCompanyNameFormProviderSpec.scala

```diff
@@ -49,5 +53,11 @@ class RegisteredCompanyNameFormProviderSpec extends StringFieldBehaviours {
       fieldName,
       requiredError = FormError(fieldName, requiredKey)
     )
+
+    "must not bind invalid Registered Company Name" in {
+      val invalidRegisteredCompanyName = "Invalid%comp@ny name?*]"
```

Comment:
Which part of this makes it invalid? Did James' response specify differently to our assumptions that company names may have special characters